test/e2e/deploy/volumeleaks.go

```diff
@@ -48,8 +48,16 @@ func GetHostVolumes(d *Deployment) map[string][]string {
 		// Intentional Output instead of CombinedOutput to dismiss warnings from stderr.
 		// lvs may emit lvmetad-related WARNING msg which can't be silenced using -q option.
 		out, err := ssh.Output()
-		if err != nil && os.IsNotExist(err) {
-			break
+		if err != nil {
+			if os.IsNotExist(err) {
+				// ssh wrapper does not exist: all node handled.
+				break
+			}
+			ginkgo.By(fmt.Sprintf("ssh on node %d returned error '%s', retry once", worker, err.Error()))
```

Comment:
Why once? Retrying for a certain time may be more predictable.

Reply:
The logic behind "once" is that, we likely get hit by a random problem, and these usually don't occur multiple in row, so re-trying more than once will not change the result. The problem likely is some resource shortage, like IO slowness, connectivity issue, timeout etc.
Although, I think it would be good idea to add some delay before retry, because if the bad condition gets over in time, we should give some time.
Let's also note that it is very unlikely to hit  the path we try to test here in regular CI run. In CI we can only verify we dont break it more.

Reply:
If it's simply about probability, then retrying twice is better than retrying once and does not hurt if the assumption is that we never have to try more than once...